inst/extdata/dc/schema-template.json

```diff
@@ -1,5 +1,9 @@
 {
-    "_note": "Fields like this starting with underscores are comments. Please delete this field, and all fields in this template except for those you wish to use for your deposit. This template may NOT be used in anything like this default form to construct a 'deposits' client. Many values require editing to comply with expected formats described throughout, such as dates or fields expected to accord with some fixed vocabulary, or modification from text descriptions to JSON objects. Please refer to the main 'dc/schema' file of the 'deposits' package for full details.",
```

Comment:
This comment line should remain here though, right? It is intended to demonstrate exactly the kind of functionality this PR would enable

Reply:
I have split it up[ in multiple `_note` properties so it is easier to read. The info is the same, only the part that this field should be deleted is removed.